LICENSE.md

```diff
@@ -70,9 +71,9 @@ their own licenses:
 
 The following components of Julia's standard library have separate licenses:
 
-- base/fftw.jl (see [FFTW](http://fftw.org/doc/License-and-Copyright.html))
```

Comment:
Yes, but it should still say something about the FFTW dependency if you call the fft etc. routines